pkg/ddl/backfilling_dist_scheduler.go

```diff
@@ -594,11 +594,18 @@ func getRangeSplitter(
 			logger.Warn("fail to get region split keys and size", zap.Error(err))
 		}
 	}
-
+	nodeRc := handle.GetNodeResource()
+	rangeSize, rangeKeys := external.CalRangeSize(nodeRc.TotalMem/int64(nodeRc.TotalCPU), regionSplitSize, regionSplitKeys)
+	logutil.DDLIngestLogger().Info("split kv range with split size and keys",
+		zap.Int64("region-split-size", regionSplitSize),
+		zap.Int64("region-split-keys", regionSplitKeys),
+		zap.Int64("range-size", rangeSize),
+		zap.Int64("range-keys", rangeKeys),
+	)
 	// no matter region split size and keys, we always split range jobs by 96MB
```

Comment:
Does this comment still take effect？

Reply:
https://github.com/pingcap/tidb/pull/59870/commits/7c6a082ddb39f4c08089950954e3c73d73ab8718